addons/account/models/account_account.py

```diff
@@ -135,6 +138,9 @@ def _check_account_type_unique_current_year_earning(self):
                                help="If set, this account will belong to Non Trade Receivable/Payable in reports and filters.\n"
                                     "If not, this account will belong to Trade Receivable/Payable in reports and filters.")
 
+    # Form view: show code mapping tab or not
+    display_mapping_tab = fields.Boolean(default=lambda self: len(self.env.user.company_ids) > 1, store=False)
```

Comment:
Does this really work? Why would the default function be called when opening an existing record?

Shouldn't it be a compute?

Reply:
It does actually work though - e.g. Mitchell Admin has access to both companies -> the Mapping tab appears
![image](https://github.com/user-attachments/assets/3b2cdc8e-e4fa-4eae-8e9d-73d4c011e13c)

Mark Demo only has access to one company -> it disappears
![image](https://github.com/user-attachments/assets/f9454d80-9733-4706-aec7-f8e053d684ce)

Reply:
I believe this is why:
https://github.com/odoo/odoo/blob/2b87effeb461acddf327884dd72c00ee22a4c9aa/odoo/fields.py#L1324